### PR TITLE
doc: Add mlinux gateway guide

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add multitech mLinux gateway guide

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in `README.md`.
- [x] Style Guidelines: Documentation obeys style guidelines in `README.md`.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: doc/content/gateways/multitechconduit-mlinux/_index.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+Reboot your gateway to enable the new network configuration, and now you can connect your gateway to your router and finish the configuration for {{% tts %}}.
+
+## Configuration using a Terminal
+
+### Downloading Global_conf.json
+
+Firstly, you will need to generate a `global_conf.json` file required to add a custom channel plan to your gateway. Follow the steps below to generate the required json file.
+
+> **Warning:** The manual configuration of the frequency plan - required to make it work on the The Things (Enterprise) Stack - may result in a mismatch for some of the frequency bands used for the Multitech Gateway.
```
Comment (laurensslats) [inline doc/content/gateways/multitechconduit-mlinux/_index.md:79]:
```suggestion
> **Warning:** The manual configuration of the frequency plan - required to make it work on the The Things Stack - may result in a mismatch for some of the frequency bands used for the Multitech Gateway.
```

File: doc/content/gateways/multitechconduit-mlinux/_index.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+The Gateway Configuration Server can be used to generate a proper `global_conf.json` configuration file for your gateway. You will need a Gateway API key with the `View gateway information` right enabled. The instructions for the same can be found in the [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}) guide.
```
Comment (laurensslats) [inline doc/content/gateways/multitechconduit-mlinux/_index.md:81]:
```suggestion
The Gateway Configuration Server can be used to generate a proper `global_conf.json` configuration file for your gateway. You will need a Gateway API key with the `View gateway information` right enabled. The instructions for creating the API key can be found in the [Adding Gateways]({{< ref "/gateways/adding-gateways" >}}) guide.
```

File: doc/content/gateways/multitechconduit-mlinux/_index.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+Open the command prompt in Windows or any Linux terminal to run a curl command (as shown below) to generate the required `global_conf.json` file in your current working directory.
+
+Make sure you replace `thethings.example.com` with your server address:
```
Comment (laurensslats) [inline doc/content/gateways/multitechconduit-mlinux/_index.md:85]:
```suggestion
Make sure you replace `thethings.example.com` with your server address and `{GATEWAY_ID}` with your Gateway ID.
```

File: doc/content/gateways/multitechconduit-mlinux/_index.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+```bash
+$ curl -XGET \
+    "https://thethings.example.com/api/v3/gcs/gateways/{GATEWAY_ID}/semtechudp/global_conf.json" \
+    -H "Authorization: Bearer {GTW_API_KEY}" > ~/global_conf.json
+```
+
+> Note: Replace the required fields in the above command and run it.
+
```
Comment (laurensslats) [inline doc/content/gateways/multitechconduit-mlinux/_index.md:94]: Can we add the instructions (or reference) for downloading the global_conf.json file from the console?

**Review by laurensslats — CHANGES_REQUESTED** [review]
Why don't you merge it with the Multitech AEP page?

I'd suggest to have 1 page "Multitech Conduit" and 2 sub-page:
Conduit mLinux
Conduit AEP

**Comment by benolayinka** [comment]
> Why don't you merge it with the Multitech AEP page?
> 
> I'd suggest to have 1 page "Multitech Conduit" and 2 sub-page:
> Conduit mLinux
> Conduit AEP

Because it will break documentation formatting if every gateway has depth 1, but multitech has depth 2 with two sub gateways

**Comment by htdvisser** [comment]
I think this one had a bit of a bad rebase or something. I can't imagine there need to be so many changes to add an mlinux gateway guide.

**Comment by laurensslats** [comment]
> I think this one had a bit of a bad rebase or something. I can't imagine there need to be so many changes to add an mlinux gateway guide.

Yeah, I messed up something. Should be reverted now.

File: doc/content/gateways/multitechconduit-mlinux/_index.md (hunk 4 above)
Comment (laurensslats) [reply doc/content/gateways/multitechconduit-mlinux/_index.md:94]: @benolayinka did you look into this?

File: doc/content/gateways/multitechconduit-mlinux/_index.md (hunk 3 above)
Comment (laurensslats) [reply doc/content/gateways/multitechconduit-mlinux/_index.md:85]: @benolayinka this is also not added yet, right?

**Review by laurensslats — APPROVED** [review]

**Pull request merged**